commandapi-core/src/main/java/dev/jorel/commandapi/executors/CommandArguments.java

```diff
@@ -179,6 +191,111 @@ public Optional<Object> getOptional(String nodeName) {
 		}
 		return Optional.of(argsMap.get(nodeName));
 	}
+
+	/**
+	 * Returns a raw argument by its position
+	 *
+	 * @param index The position of this argument
+	 * @return An argument which is placed at the given index, or {@code null} if the provided index does not point to an argument.
+	 */
+	@Nullable
+	public String getRaw(int index) {
+		if (rawArgs.length <= index) {
+			return null;
+		} else {
+			return rawArgs[index];
+		}
+	}
+
+	/**
+	 * Returns a raw argument by its node name
+	 *
+	 * @param nodeName The node name of this argument. This was set when initializing an argument
+	 * @return A raw argument which has the given node name. Can be null if <code>nodeName</code> was not found.
+	 */
+	@Nullable
+	public String getRaw(String nodeName) {
+		return rawArgsMap.get(nodeName);
+	}
+
+	/**
+	 * Returns a raw argument by its index
+	 *
+	 * @param index The position of this argument
+	 * @param defaultValue The String returned if the raw argument is not existent
+	 * @return A raw argument which is placed at the given index, or the provided default value
+	 */
+	public String getOrDefaultRaw(int index, String defaultValue) {
+		if (rawArgs.length <= index) {
+			return defaultValue;
+		} else {
+			return rawArgs[index];
+		}
+	}
+
+	/**
+	 * Returns a raw argument by its node name
+	 *
+	 * @param nodeName     The node name of this argument. This was set when initializing an argument
+	 * @param defaultValue The String returned if the raw argument was not found.
+	 * @return A raw argument with the specified node name or the provided default value
+	 */
+	public String getOrDefaultRaw(String nodeName, String defaultValue) {
+		return rawArgsMap.getOrDefault(nodeName, defaultValue);
+	}
+
+	/**
+	 * Returns a raw argument by its index
+	 *
+	 * @param index The position of this argument
+	 * @param defaultValue The String returned if the raw argument is not existent
+	 * @return A raw argument which is placed at the given index, or the provided default value
+	 */
+	public String getOrDefaultRaw(int index, Supplier<String> defaultValue) {
+		if (rawArgs.length <= index) {
+			return defaultValue.get();
+		} else {
+			return rawArgs[index];
+		}
+	}
+
+	/**
+	 * Returns a raw argument by its node name
+	 *
+	 * @param nodeName     The node name of this raw argument. This was set when initializing an argument
+	 * @param defaultValue The String returned if the raw argument was not found.
+	 * @return A raw argument with the specified node name or the provided default value
+	 */
+	public String getOrDefaultRaw(String nodeName, Supplier<String> defaultValue) {
+		return rawArgsMap.getOrDefault(nodeName, defaultValue.get());
+	}
+
+	/**
+	 * Returns an <code>Optional</code> holding the raw argument by its index
```

Comment:
`Optional` shouldn't be wrapped in `<code>`, it should be in a `@link` instead (e.g. `{@link Optional}`)